docs/releases/v13/version-v13.0.0.md

```diff
@@ -68,15 +68,17 @@ Collections introduce a powerful new way to organize and curate records within y
 Collections provide dedicated pages showing all records matching specific criteria.
 ///
 
-**Hierarchical organization**
+*Hierarchical organization*
+
```

Comment:
The `**` made this header look like the **Collection** header even if it's below it. Italicizing and making sure to add new line makes it clearer that it is under the **Collections** section.